Pegasus/sources/Geometry.cpp

```diff
@@ -484,7 +472,8 @@ glm::dvec3 intersection::gjk::NearestSimplex(Simplex& simplex)
     {
         return ::NearestSimplexLineSegment(simplex);
     }
-    else if (3 == simplex.size)
+    
+    if (3 == simplex.size)
```

Comment:
Why not else if? There is no way to both get inside both `if`s since there is a `return` inside

Reply:
Well, is not it the definition of a redundant `else`?